### PR TITLE
Consistency

### PR DESCRIPTION
**Pull request by amuolo**
This PR resolves the ticket #136

### CODE REVIEW

File: ifrs17-template/Files/TransactionalData/Actuals_CH_2020_12.csv [hunk 1]
```diff
@@ -1,55 +1,55 @@
...
-DT3.1,CF,AEA,,-5
-DT4.1,CF,PR,,400
-DT4.1,CF,NIC,,-280
-DT4.1,CF,ACA,,-10
-DT4.1,CF,AEA,,-5
-DT1.5,CF,PR,,400
-DT1.5,CF,NIC,,-280
-DT1.5,CF,ACA,,-10
-DT1.5,CF,AEA,,-5
-DT1.5,CF,ReceivablePR,,-10
```
Comment (dtrzesniak) [inline ifrs17-template/Files/TransactionalData/Actuals_CH_2020_12.csv:46]: You should also remove these mapping keywords from the dimensions.csv, just keep it at the corresponding client projects

File: ifrs17-template/Files/TransactionalData/Actuals_CH_2020_12.csv [hunk 2]
```diff
@@ -1,55 +1,55 @@
...
+DT4.1,CF,ACA,A,,-10,,
+DT4.1,CF,AEA,A,,-5,,
+DT1.5,CF,PR,A,,400,,
+DT1.5,CF,NIC,A,,-280,,
+DT1.5,CF,ACA,A,,-10,,
+DT1.5,CF,AEA,A,,-5,,
+DT1.5,CF,PR,OA,,-10,,OA
+DTR1.3,CF,PR,A,,-200,,
+DTR1.3,CF,NIC,A,,140,,
+DTR1.3,CF,PR,OA,,7.5,,AA
```
Comment (dtrzesniak) [inline ifrs17-template/Files/TransactionalData/Actuals_CH_2020_12.csv:49]: What is this extra "AA"?

Comment (dtrzesniak) [reply ifrs17-template/Files/TransactionalData/Actuals_CH_2020_12.csv:49]: Also other cases

**Review by dtrzesniak — APPROVED** [review]

File: ifrs17-template/Files/TransactionalData/Actuals_CH_2020_12.csv (hunk 2 above)
Comment (amuolo) [reply ifrs17-template/Files/TransactionalData/Actuals_CH_2020_12.csv:49]: my bad, I added these 2 by mistake, now canceled them. Thanks a lot!

File: ifrs17-template/Files/TransactionalData/Actuals_CH_2020_12.csv (hunk 1 above)
Comment (amuolo) [reply ifrs17-template/Files/TransactionalData/Actuals_CH_2020_12.csv:46]: ok got it. Thanks, I will now do so.

Comment (amuolo) [reply ifrs17-template/Files/TransactionalData/Actuals_CH_2020_12.csv:46]: done

File: ifrs17-template/Files/TransactionalData/Actuals_CH_2020_12.csv (hunk 2 above)
Comment (amuolo) [reply ifrs17-template/Files/TransactionalData/Actuals_CH_2020_12.csv]: done

**Pull request merged**